From: Pixel <pixel@mandrakesoft.com>
Subject: Re: NTFS-fs warning
To: Juan Quintela <quintela@mandrakesoft.com>
Cc: Pablo Saratxaga <pablo@mandrakesoft.com>, kernel@mandrakesoft.com
Date: 21 Feb 2003 18:57:46 +0100

Juan Quintela <quintela@mandrakesoft.com> writes:

> pixel> could "iocharset=" be handled correctly for ntfs? it is even written
> pixel> in mount(8) that iocharset= is handled by ntfs.
> 
> will look, I have never touched the ntfs code, that was jeff garzik
> part :( 

[2. text/x-patch; ntfs.patch]

--- linux-2.4.20/fs/ntfs/super.c.pix	2003-02-21 18:56:58.000000000 +0100
+++ linux-2.4.20/fs/ntfs/super.c	2003-02-21 18:56:34.000000000 +0100
@@ -154,11 +154,13 @@
 			ntfs_warning(vol->sb, "Ignoring obsolete option %s.",
 					p);
 		else if (!strcmp(p, "nls") || !strcmp(p, "iocharset")) {
+#if 0
 			if (!strcmp(p, "iocharset"))
 				ntfs_warning(vol->sb, "Option iocharset is "
 						"deprecated. Please use "
 						"option nls=<charsetname> in "
 						"the future.");
+#endif
 			if (!v || !*v)
 				goto needs_arg;
 use_utf8:

?
